### PR TITLE
[incubator/kube-spot-termination-notice-handler] Add Support for Tolerations

### PR DESCRIPTION
**Pull request by stevenpall**
**What this PR does / why we need it:**
- Add tolerations section to daemonset template
- Add tolerations section to values file (with example configs)

**Which issue this PR fixes** (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close that issue when PR gets merged):
- N/A

**Special notes for your reviewer:**
- N/A

### CODE REVIEW

File: incubator/kube-spot-termination-notice-handler/templates/daemonset.yaml [hunk 1]
```diff
@@ -46,3 +46,9 @@ spec:
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 8 }}
     {{- end }}
+{{- if .Values.tolerations }}
+      tolerations:
+      - {{- range $key, $value := .Values.tolerations }}
+{{- end }}
+{{ toYaml .Values.tolerations | indent 8 }}
+{{- end }}
```
Comment (unguiculus) [inline incubator/kube-spot-termination-notice-handler/templates/daemonset.yaml]: Please change as follows:

```yaml
    {{- if .Values.tolerations }}
      tolerations:
{{ toYaml .Values.tolerations | indent 8 }}
    {{- end }}

File: incubator/kube-spot-termination-notice-handler/values.yaml [hunk 2]
```diff
@@ -40,3 +40,9 @@ serviceAccount:
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name:
+
+tolerations: {}
```
Comment (unguiculus) [inline incubator/kube-spot-termination-notice-handler/values.yaml]: `tolerations: []`

File: incubator/kube-spot-termination-notice-handler/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Watch and action AWS spot termination events
 name: kube-spot-termination-notice-handler
-version: 0.2.1
+version: 0.2.2
```
Comment (unguiculus) [inline incubator/kube-spot-termination-notice-handler/Chart.yaml]: Bump minor version for a new feature.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by stevenpall** [comment]
@unguiculus Let me know if that looks good now!

**Review by unguiculus — APPROVED** [review]

**Pull request merged**